src/components/mobile/mobile-close-aside.vue

```diff
@@ -1,16 +1,43 @@
 <template>
-  <BaseIdModalClose modalId="aside-modal" class="x-button x-button--pill">
-    <FiltersIcon />
-    <span>{{ $t('toggleAside.hideAside') }}</span>
-  </BaseIdModalClose>
+  <div
+    class="
+      x-list x-list--horizontal x-list--gap-05
+      x-border-color--neutral-10
+      x-padding--05 x-padding--top-03 x-padding--bottom-04
+    "
+  >
+    <ClearFilters
+      v-slot="{ selectedFilters }"
+      class="
+        x-button--secondary x-button--round
+        x-padding--05 x-padding--top-04 x-padding--bottom-04
+        x-uppercase
+        x-list__item--expand
+      "
+      :alwaysVisible="false"
+    >
+      {{ $t('selectedFilters.clear', { selectedFiltersNumber: selectedFilters.length }) }}
```

Comment:
Remove the white spaces in the translations:

`"clear": "Clear {selectedFiltersNumber} filters"`